api/go.mod

```diff
@@ -1,27 +1,28 @@
 module kubevirt.io/controller-lifecycle-operator-sdk/api
 
-go 1.17
+go 1.22.0
```

Comment:
https://github.com/kubevirt/kubevirt/pull/12130 is a clearer format IMO

Reply:
Not sure it matters much given this is a library repo but can we stick with this for now until we have a consensus in kubevirt?